BigBaseV2/src/views/vehicle/view_lsc.cpp

```diff
@@ -518,23 +538,25 @@ namespace big
 			ImGui::SetNextItemWidth(212);
 			if (ImGui::ColorPicker3("Custom VehColor", color, ImGuiColorEditFlags_NoAlpha | ImGuiColorEditFlags_NoDragDrop | ImGuiColorEditFlags_NoOptions | ImGuiColorEditFlags_DisplayRGB | ImGuiColorEditFlags_DisplayHex))
 			{
-				g_fiber_pool->queue_job([] {
-					switch (color_to_change)
-					{
-					case 0:
-						VEHICLE::SET_VEHICLE_CUSTOM_PRIMARY_COLOUR(player_vehicle, (int)(color[0] * 255), (int)(color[1] * 255), (int)(color[2] * 255));
-						break;
-					case 1:
-						VEHICLE::SET_VEHICLE_CUSTOM_SECONDARY_COLOUR(player_vehicle, (int)(color[0] * 255), (int)(color[1] * 255), (int)(color[2] * 255));
-						break;
-					case 2:
-						VEHICLE::SET_VEHICLE_TYRE_SMOKE_COLOR(player_vehicle, (int)(color[0] * 255), (int)(color[1] * 255), (int)(color[2] * 255));
-						break;
-					case 3:
-						VEHICLE::SET_VEHICLE_NEON_LIGHTS_COLOUR_(player_vehicle, (int)(color[0] * 255), (int)(color[1] * 255), (int)(color[2] * 255));
-						break;
-					}
-				});
+				color_rgb[0] = (int)(color[0] * 255);
+				color_rgb[1] = (int)(color[1] * 255);
+				color_rgb[2] = (int)(color[2] * 255);
+
+				switch (color_to_change)
+				{
+				case 0:
+					VEHICLE::SET_VEHICLE_CUSTOM_PRIMARY_COLOUR(player_vehicle, color_rgb[0], color_rgb[1], color_rgb[2]);
+					break;
+				case 1:
+					VEHICLE::SET_VEHICLE_CUSTOM_SECONDARY_COLOUR(player_vehicle, color_rgb[0], color_rgb[1], color_rgb[2]);
+					break;
+				case 2:
+					VEHICLE::SET_VEHICLE_TYRE_SMOKE_COLOR(player_vehicle, color_rgb[0], color_rgb[1], color_rgb[2]);
+					break;
+				case 3:
+					VEHICLE::SET_VEHICLE_NEON_LIGHTS_COLOUR_(player_vehicle, color_rgb[0], color_rgb[1], color_rgb[2]);
```

Comment:
Calling natives outside of script threads is detected.

Use the fiber pool please.